src/example/c_local_example.c

```diff
@@ -0,0 +1,248 @@
+/***********************************************************************
+/
+/ Example executable using libgrackle
+/
+/
+/ Copyright (c) 2013, Enzo/Grackle Development Team.
+/
+/ Distributed under the terms of the Enzo Public Licence.
+/
+/ The full license is in the file LICENSE, distributed with this 
+/ software.
+************************************************************************/
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <math.h>
+#include <string.h>
+#include <unistd.h>
+
+#include <grackle.h>
+
+#define mh     1.67262171e-24   
+#define kboltz 1.3806504e-16
+
+int main(int argc, char *argv[])
+{
+
+  /*********************************************************************
+  / Initial setup of units and chemistry objects.
+  / This should be done at simulation start.
+  *********************************************************************/
+
+  // Set initial redshift (for internal units).
+  double initial_redshift = 0.;
+
+  // Enable output
+  grackle_verbose = 1;
+
+  // First, set up the units system.
+  // These are conversions from code units to cgs.
+  code_units my_units;
+  my_units.comoving_coordinates = 0; // 1 if cosmological sim, 0 if not
+  my_units.density_units = 1.67e-24;
+  my_units.length_units = 1.0;
+  my_units.time_units = 1.0e12;
+  my_units.a_units = 1.0; // units for the expansion factor
+  // Set expansion factor to 1 for non-cosmological simulation.
+  my_units.a_value = 1. / (1. + initial_redshift) / my_units.a_units;
+  set_velocity_units(&my_units);
+
+  // Second, create a chemistry object for parameters.  This needs to be a pointer.
```

Comment:
```suggestion
  // Second, create a chemistry object for parameters.
```
It doesn't actually NEED to be a pointer

Reply:
Thanks for the suggestions! I have made the changes and will push shortly.